packages/backend/src/auth/github.strategy.ts

```
@@ -0,0 +1,27 @@
+      callbackURL: configService.get('GITHUB_CALLBACK_URL')!,
+      scope: ['user:email'],
+    });
+  }
+
+  async validate(accessToken: string, refreshToken: string, profile: Profile) {
+    const { id, username, emails } = profile;
+
+    return {
+      githubId: id,
+      username,
+      email: emails && emails[0].value,
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)